util.js

```diff
@@ -74,7 +74,7 @@ function trimSlashes (str) {
 }
 
 function cachedPrebuild (url) {
-  const digest = crypto.createHash('md5').update(url).digest('hex').slice(0, 6)
+  const digest = crypto.createHash('sha512').update(url).digest('hex').slice(0, 6)
```

Comment:
If the reason for not using an MD5 hash under the rules of FIPS is its predictability and therefore increased chance of collision then the subsequent use of `slice(0, 6)` negates hash choice anyway. :sweat_smile:

Reply:
Well, it doesn't much matter in the end. This code will not be running as part of the system, only during build.
At the same time, node unfortunately, does not have an equivalent of `usedforsecurity` parameter from python's [hashlib](https://docs.python.org/3/library/hashlib.html)